matrix/inverse_of_matrix.py

```diff
@@ -0,0 +1,54 @@
+from decimal import *
+
+
+def inverse_of_matrix(matrix):

```

Comment:
Type hints, please? This function should return a matrix.  If there is none then it should either raise a ValueError or return an empty list.